### PR TITLE
MO-576 set Rails Timezone to London time 

### PR DESCRIPTION
**Pull request by starswan**
so that times don't come out in GMT

### CODE REVIEW

File: spec/features/allocation_history_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-feature 'Allocation History' do
+xfeature 'Allocation History' do
   let(:probation_pom) do
```
Comment (Katy600) [inline spec/features/allocation_history_spec.rb]: Is this feature test pending?

File: spec/features/allocation_history_spec.rb [hunk 2]
```diff
@@ -222,7 +222,15 @@
               expect(page).to have_css(key, text: val)
             end
           end
+        end
+      end
+
+      it 'has a Pentonville section', :js do
+        # 2nd Prison - Pentonville. This contains 7 events
```
Comment (Katy600) [inline spec/features/allocation_history_spec.rb:229]: Now you have split the test into 2 does this test need this comment now? Is this now covered in the first test?

**Review by Katy600 — APPROVED** [review]
Hi Stephen, 
I have tried really hard to find things to say. I think my comments are mostly about trying to understand the tests but these are not actually your tests! So feel free to ignore them.

File: spec/features/allocation_history_spec.rb (hunk 1 above)
Comment (starswan) [reply spec/features/allocation_history_spec.rb]: reverted

**Pull request merged**